src/index.html

```diff
@@ -7,8 +7,71 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >  
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav__wrap">
+      <a href="#home">
+        <img 
+          class="img_settings"
+          src="images/logo.png"
+          alt="Moyo Logo"
+        >
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a class="is-active nav__link" href="http://apple.html">
```

Comment:
```suggestion
            <a class="is-active nav__link" href="#apple">
```
its should be a dummy links, everywhere